src/main/java/blackjack/domain/state/running/Hit.java

```diff
@@ -0,0 +1,32 @@
+package blackjack.domain.state.running;
+
+import blackjack.domain.Response;
+import blackjack.domain.cards.Card;
+import blackjack.domain.cards.Hand;
+import blackjack.domain.state.State;
+import blackjack.domain.state.finished.Stay;
+import blackjack.domain.state.hitstrategy.HitStrategy;
+
+public class Hit extends Running {
+
+    private final HitStrategy hitStrategy;
+
+    public Hit(Hand hand, HitStrategy hitStrategy) {
+        super(hand);
+        this.hitStrategy = hitStrategy;
+    }
+
+    @Override
+    public State draw(Card card) {
+        getHand().addCard(card);
+        return hitStrategy.moveToState(getHand());
+    }
+
+    @Override
+    public State moveStateByResponse(Response response) {
+        if (response == Response.NEGATIVE) {
```

Comment:
```suggestion
        if (response.isNegative()) {
```

Enum 같은 경우도 직접 비교보다 메시지로 확인할 수 있어요. 
다른 Enum에도 적용할 곳이 있는지 같이 확인해봐도 좋겠네요 :)